### DISCUSSION TITLE
Behringer X-Touch weird behavior

### DISCUSSION THREAD

**Discussion by Omnichordman** [bjoluc/cubase-mcu-midiremote#45] [Q&A] [answered]
Hi,

since a coupe of weeks my X-Touch has a strange problem: With some of my Projects all of a sudden the Encoder Assign buttons don´t lite up anymore. Also the Loop-Button does not stay on. I have to "confirm" any of my acctions here by clicking the select button after every Encoder Assign button press or when I want to keep the dial in Zoom-Mode. 
This happens with all of my new projects, unless I start from as template, where the X-Touch works as expected. I tried to re-import my files into a new template, and while that sometimes works, it can happen that after I save, the X-Touch loses it again...

I am not a programmer and at the end of my rope here. Can anybody help?
I run Cubase 12.0.70.464 and I used the latest X-touch js file.

thanks in advance

best regards 
Joker Nies

**Comment by bjoluc** [comment, marked as answer]
Hi @Omnichordman,
what you describe sounds like a Cubase issue that's typically fixed by [disabling and re-enabling the controller script](https://github.com/bjoluc/cubase-mcu-midiremote?tab=readme-ov-file#how-to-disable-and-re-enable-a-controller-script) – does that change anything?

**Reply by Omnichordman** [reply]
Thanks so much for your instant reply.
This works like a charm! Finally I can use my X-Touch again without any hassle. 
Your script really made that controller my workhorse!

best,
Onichordman

**Comment by ebz1989** [comment]
This happens to me also, sometimes it is fixed just doing what you said about disabling and re-enabling the script. But in some projects, this doesn't work at all. There is no way to make it work properly again.

**Comment by HeikoGrein** [comment]
Hello! I hope that this issues fits into this conversation here too?
I am on Cubase 14 Pro (Windows 10) and downloaded and installed the latest script from GitHub - bjoluc/cubase-mcu-midiremote: Cubase MIDI Remote Scripts for DAW Controllers using the MCU Protocol. The scipt is visisble and active.
No other device is installed or connected.  My X-Touch One is running on the latest FW 1.10. The MCU protocol is set to MCU - CU. The X-Touch One receives and displays data from the DAW. The Barcounter (Display) of the unit is also following the DAW.
BUT if I try to use any transport function or the fader or the X-Touch it does not “talk” too/ correspond with any fader of the DAW, though.
disabling and re-enabling the controller does not change anything.

Any help is welcome.
Thanks in advance.

**Reply by ebz1989** [reply]
Hey man. I don't know what could be happening there. , re-check the devices menu and go to the remote devices where you find by default a "mackie control", vst quick controls. Maybe any of them have taken the Behringer.

**Reply by HeikoGrein** [reply]
@ebz1989 Thanks for the swift reply. No other device installed, and also all relevant pop-up menus are showing “disconnected”. It sometimes works, but it is not working reliable, unfortunately. I tried several new starts of Cubase and my computer too.

**Reply by HeikoGrein** [reply]
@bjoluc Hello Bjoern, Did you come across the issue, pls see above before? Thx in advance. Heiko

**Comment by esazhin** [comment]
This solution might be relevant to the original discussion about encoder buttons behavior: https://github.com/bjoluc/cubase-mcu-midiremote/issues/63

**Reply by HeikoGrein** [reply]
@esazhin Thank you! Will test the solution and report feedback. Thx Heiko
